src/DI/Loader/DoctrineAnnotationLoader.php

```diff
@@ -125,14 +125,14 @@ protected function analyseClass($class)
 	protected function acceptController(ClassType $class)
 	{
 		// Has class annotation @Controller?
-		if ($class->hasAnnotation('Controller')) return TRUE;
+		if ($this->createReader()->getClassAnnotation($class, \Apitte\Core\Annotation\Controller\Controller::class)) return TRUE;
```

Comment:
createReader should be getReader in my opinion - it doesnt always create a new instance

Reply:
Good point. Shall I include it in this PR @f3l1x?

Reply:
Yep, good point.